policies/gitlab/repository.rego

```diff
@@ -54,11 +47,11 @@ project_has_too_many_admins {
 #   remediationSteps: [Make sure you have owner permissions, Go to the project's settings page, Enter "General" tab, Under "Visibility, project features, permissions", Toggle off "Forks"]
 #   severity: LOW
 #   threat: Forked repositories may leak important code assets or sensitive secrets embedded in the code to anyone outside your organization, as the code becomes publicy-accessible
-default forking_allowed_for_repository = false
+default forking_allowed_for_repository = true
 
-forking_allowed_for_repository {
-	input.public == false
-	input.forking_access_level == "enabled"
+forking_allowed_for_repository = false {
+	input.public
```

Comment:
this policy applies to private repositories

Reply:
fixed